internal/updater/check.go

```diff
@@ -98,6 +99,9 @@ func AutoUpdate(pjPath string, out output.Outputer) (updated bool, resultVersion
 		if os.IsPermission(errs.InnerError(err)) {
 			out.Error(locale.T("auto_update_permission_err"))
 		}
+		if errors.As(err, new(*osutils.AlreadyLockedError)) {
+			out.Notice(locale.T("auto_update_already_updating"))
```

Comment:
Auto update is implicit. The user did not ask to be updated. Showing them this notice would only add friction.

Let's instead ensure that the error is logged if it isn't an error we can recognize, but otherwise not expose this to the user.

Reply:
The error will be logged here if it is not a permissions error, in which case the error is reported to the user. Do we want to remove this behaviour as well?

I've left the existing log line in `updater.go` as it could provide some value information in debugging.

Reply:
Yeah let's not log it (as an error anyway), since this is "expected" behaviour.

Reply:
Okay changed it to a debug log